### DISCUSSION TITLE
Allow StatusEffect to add and remove item tags, rather than just overwriting them.

### DISCUSSION THREAD

**Discussion by zhu-rengong** [FakeFishGames/Barotrauma#14635] [Code questions and modding support]
There are many places that may modify an item's tags for checking if it has certain tags by other parts of the code, but each modification removes all previous addtional tags, which makes tag management very difficult.

**Comment by UnhappyWithMyUsername** [comment]
Yeah I've suggested the same thing a while ago here:
https://github.com/FakeFishGames/Barotrauma/discussions/13214
Would be really great if they added such a functionality.
